src/main/java/com/internet/mixshop/controller/DeleteUserController.java

```diff
@@ -0,0 +1,23 @@
+package com.internet.mixshop.controller;
+
+import com.internet.mixshop.lib.Injector;
+import com.internet.mixshop.service.UserService;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class DeleteUserController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("com.internet.mixshop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        String userId = req.getParameter("id");
+        Long id = Long.valueOf(userId);
+        userService.delete(id);
```

Comment:
let's delete cart here as well

Reply:
fix this comments please, rest seem to be good

Reply:
sorry, fixed